### PR TITLE
JSM: Update Modularize Script to Produce Usable Modules

### PR DESCRIPTION
**Pull request by gkjohnson**
Updates the `modularize.js` script to automatically perform some of the fixes made in https://github.com/mrdoob/three.js/pull/15572.

Some of the notable changes that this results in:

### Update 1
```js
MapControls.prototype.constructor = THREE.MapControls;
```
to
```js
MapControls.prototype.constructor = MapControls;
```

### Update 2
```js
export { MapControls }
```
to
```js
export { MapControls };
```

### Update 3

This one may be undesirable but it's a side effect of the first fix without getting more sophisticated in excluding strings from the regex replacement.

```js
console.warn( 'THREE.MapControls: .dynamicDampingFactor has been renamed. Use .dampingFactor instead.' );
```
to
```js
console.warn( 'MapControls: .dynamicDampingFactor has been renamed. Use .dampingFactor instead.' );
```

### CODE REVIEW

**Comment by donmccurdy** [comment]
Unless (3) is a change we like anyway, we could revert that as each ES module becomes canonical. I don't see an easy way to avoid it in the script, meanwhile.

File: utils/modularize.js [hunk 1]
```diff
@@ -83,7 +83,7 @@ function convert( path, ignoreList ) {
 
 	var keys = Object.keys( dependencies ).sort().map( value => '\n\t' + value ).toString();
 	var imports = `import {${keys}\n} from "../../../build/three.module.js";`;
-	var exports = `export { ${className} }`;
+	var exports = `export { ${className} };`;
```
Comment (donmccurdy) [inline utils/modularize.js]: Could you add a closing `\n` here as well?

**Comment by gkjohnson** [comment]
Ping @mrdoob.

Any thoughts on this? Should anything change? At the moment if the `modularize.js` script is run again the Control scripts will be broken without manual changes.

**Review by Mugen87 — APPROVED** [review]

**Comment by donmccurdy** [comment]
Note that there are ~6 open pull requests for converting things to ES modules, several of which may conflict: https://github.com/mrdoob/three.js/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aopen+in%3Atitle+JSM

Updating the modularize script seems like the first priority, since many of the other PRs will be affected by that.

**Comment by Mugen87** [comment]
Merging so the control modules are finally usable...

**Comment by mrdoob** [comment]
Thanks!

**Comment by mrdoob** [comment]
I updated the regex to avoid changing the `console.log()`s: https://github.com/mrdoob/three.js/commit/822530be589bd5d0c68e2a9197cd3d7901f04c9a

**Pull request merged**